lib/business_time/currency.rb

```diff
@@ -1,26 +1,27 @@
 module BusinessTime
   module Currency
+
     # @return [Array] the list of currencies
     # @param [String+]
     def args(*currency)
-      return [] if currency.empty?
+      return Set.new if currency.empty?
```

Comment:
Totally unrelated to this task, but I have to mention this. You have likely 10 times repeated assignment `Set.new` to something. Imagine tomorrow we’ll decide to swicth to `TenTimesFasterSet` implementation. `sed` works fine and I love `sed`, but the future-friendly solution would be:
```
module BusinessTime
  def self.Container()
    Set.new
  end
end
```
and, later:
```ruby
return Container() if currency.empty?
```